### PR TITLE
first draft of changes for enable/disable and update webhook

### PR DESCRIPTION
**Pull request by dzucker-tab**
@shinchris here is a rough draft, please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,21 +261,25 @@ Creates a new webhook for a site.  
...
 `url`   The destination URL for the webhook. The webhook destination URL must be https and have a valid certificate.
+
+`webhook-enabled-flag`   Optional, boolean. If `true` (default), the newly created webhook is enabled. If `false` then the webhook will be disabled.
   
 #### Response Code
 
-`201`  
+`201` Success.
+
+`400` Error 400127: `statusChangeReason` was provided with `isEnabled` = `true`
```
Comment (shinchris) [inline README.md:276]: This should not be part of the create endpoint

File: README.md [hunk 2]
```diff
@@ -428,7 +442,75 @@ None 
 
 ### Update a Webhook  
 
-To modify a webhook after it has been created, delete it and recreate it.
+Modify the properties of an existing webhook.
+
+#### URI
+
+```
+PUT /api/3.6/sites/<site-id>/webhooks/<webhook-id>
```
Comment (shinchris) [inline README.md:450]: Could you change the version here to 3.8?

File: README.md [hunk 3]
```diff
@@ -428,7 +442,75 @@ None 
...
+            <webhook-destination-http method="POST" url="url"/>  
+        </webhook-destination>  
+        <owner id="webhook_owner_luid" name="webhook_owner_name"/>
+    </webhook>  
+</tsResponse>
+```
+
+#### Response Headers
+
+`Location: /api/<api-version>/sites/<site-id>/webhooks/<new-webhook-id>`
```
Comment (shinchris) [inline README.md:513]: Out of curiosity, what is response header? I see it for update and create only

File: README.md (hunk 1 above)
Comment (dzucker-tab) [reply README.md:276]: will remove

File: README.md (hunk 2 above)
Comment (dzucker-tab) [reply README.md:450]: will change

File: README.md (hunk 3 above)
Comment (dzucker-tab) [reply README.md:513]: I didn't add this, but here's what I see in the response header when I  create a webhook:

**Location:** `/api/3.7/sites/a946d998-2ead-4894-bb50-1054a91dcab3/webhooks/7f60f398-a00f-4642-b0ef-66c7df5607e2`

so I am thinking it is accurately reflecting the location for addressing the webhook that has been created or updated

**Pull request merged**